### ISSUE TITLE
[Installation] @beartype's description is so big it hurts.

### ISSUE THREAD

**Issue by LittleBigGene** [beartype/beartype#203]
ERROR: Could not install requirement beartype>=0.9.1 (from flet) because of HTTP error 400 Client Error: Bad Request - The package description exceeds the maximum length of 324608 bytes.

**Comment by leycec** [comment]
**Gah!** I somehow missed this in the recent scrum to release `beartype 0.12.0`. I've... actually never even *heard* of [Flet](https://flet.dev). Looks pretty swag, though! That said, Flet isn't actually a package manager. It's a UI framework.

Which package manager were you attempting to use? `pip`? Conda? Poetry? Something else? Pretty sure those all work – but *not* certain. Perhaps something finally broke somewhere. :face_exhaling:

In any case, we might need to finally redact our package description. Admittedly, it *has* become obscenely large. Our `README.rst` currently stands at 315KB and can only grow larger with time.

**Comment by LittleBigGene** [comment]
I did "pip install flet", hoping to try it out. 

Yes, I believe it's complaining about the size of the package description.

**Comment by leycec** [comment]
**Yikes!** Thanks so much for the rapid clarification, @LittleBigGene. Great GitHub username, too. Clever puns are a fragile work of art we appreciate here.

If `pip` is indeed dying on @beartype, however, that is awful. Tragically, I can't locally replicate this:

```bash
$ pip3 install beartype==0.11.0
Defaulting to user installation because normal site-packages is not writeable
Collecting beartype==0.11.0
  Downloading beartype-0.11.0-py3-none-any.whl (702 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 702.5/702.5 kB 60.6 kB/s eta 0:00:00
Installing collected packages: beartype
Successfully installed beartype-0.11.0
```

I wonder if this might be a platform-specific issue of some sort. Are you on Windows or macOS, perchance? In any case, you're *not* wrong and I absolutely don't doubt you.

Regardless of whether I can replicate this or not, we absolutely do need to decrease the size of our package description. This is *sort of* a duplicate of issue #8, because hosting our documentation with Read The Docs (RTD) should resolve this. But let's leave this open until I actually do something. :wink:

**Comment by langfield** [comment]
Could the package description simply use a less bulky version of the `README.rst`?

**Comment by leycec** [comment]
**Totally.** You got it. I suspect I might even do the opposite:

1. Shift our entire `README.rst` onto Read the Docs (RTD) all at once. In theory, this is trivial. Our documentation already works here; it *should* work there, too. In practice, I weep sleepily.
2. Reduce the `README.rst` file hosted at this repository to just the introduction followed by a **bold-face big red link redirecting everybody to RTD.**
3. *Sloooowly* begin splitting up our documentation on RTD into multiple interlinked documents (as sanity requires).

Since our project description pulls directly from `README.rst`, this then feeds three birds with one outstretched palm. Read this five-year plan and believe my convenient promises, everyone. :palms_up_together:

**Comment by leycec** [comment]
@langfield: Oh, and thanks a heap overflow for your overwhelming generosity! Please e-mail me sometime at **Bear Bro `<leycec@gmail.com`>** when you find a spare moment and I'll happily inject your favourite URL and/or witty one-liner at the head of @beartype release changelogs.

I meant to contact you sooner about that earlier. I swear! But I couldn't find your e-mail, because my Google-stalking skills have decayed with time. (I feel old now.)

Also, **[Anki](https://github.com/langfield/ki/tree/2d6d8b4e2d8b681f43441015356825478c1c010e).** Feels so nostalgic. Can I interest in you in a [gentlemanly Japanese visual novel for inquiring minds](https://github.com/leycec/fsnrnue)? :smile_cat:

**Comment by leycec** [comment]
So. It begins... *yet again.* Die, Sphinx! Die! :face_exhaling:

> Sphinx, we face thee valiantly with great exhaustion in our hearts.

**Comment by leycec** [comment]
**Hah!** Through the power of slathering jaws slick with rabies, [@beartype *finally* forced its introduction through the Read the Docs (RTD) parser](https://beartype.readthedocs.io). Sure, most links are hideously busted. That's a given when you've neglected documentation for two years. But we will succeed. *Somehow.*

**Comment by leycec** [comment]
**Resolved by 86b79db.**

[Beartype documentation now officially lives at ReadTheDocs (RTD)](https://beartype.readthedocs.io). Our `README.rst` file has been truncated to a shadow of its former self. So this is what it feels like when bears cry.

![omg](https://media.tenor.com/JC2gzaAISiEAAAAC/sweaty-sweat.gif)
